modules/oe_theme_content_entity_contact/config/overrides/core.entity_view_display.oe_contact.oe_general.default.yml

```diff
@@ -3,22 +3,38 @@ status: true
 dependencies:
```

Comment:
As `oe_content_entity_contact` has been already released, we need to create a post update hook to modify this configuration entity.